README.md

```diff
@@ -56,11 +100,17 @@ Enables AWS Config and adds managed config rules with good defaults.
 * rds-instance-public-access-check: Checks whether the Amazon Relational Database Service (RDS) instances are not publicly accessible.
 * rds-snapshots-public-prohibited: Checks if Amazon Relational Database Service (Amazon RDS) snapshots are public.
 * rds-storage-encrypted: Checks whether storage encryption is enabled for your RDS DB instances.
+* rds-snapshot-encrypted: Checks whether Amazon Relational Database Service (Amazon RDS) DB snapshots are encrypted.
+* rds-cluster-deletion-protection-enabled: Checks whether Amazon Relational Database Service (Amazon RDS) DB snapshots are encrypted.
```

Comment:
The rule name and the description here don't seem to align. Based on the rule name, I'd assume this checks whether an RDS instance has deletion protection enabled. Based on the previous rule, with the matching description, I'm guessing maybe a bad copy/paste.

Reply:
Good catch, thank you.